@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -908,7 +921,7 @@ export class TextElementsRenderer {
 
             if (elevationProvider !== undefined && !textElement.elevated) {
                 if (!elevationMap) {
-                    this.m_viewUpdateCallback(); // Update view until elevation is loaded.
+                    this.m_isUpdatePending = true;
```

Comment:
Before we were actively requesting a MapView update to ensure there will be a follow-up frame. Now we're only passively marking the TextElementsRenderer as pending for update. Who request the next animation frame now?
Sample applies to all places where you changed the view update callback.

Reply:
Basically Mapview now checks for needed updates, as it already did for loading of the fonts and also does for other depending instances like the VisibleTiles, ExtrusionHandler, AnimationFramework, etc too.
Actually I think we should do this throughout the code and use MapView.update only as external API. 
Currently MapView is driving the RenderLoop and then we have some inconsistencies on which components are requesting updates, which are requested for updates.. and TextElementsRenderer actually had both :/